examples/dataresidency/set_region.py

```diff
@@ -0,0 +1,37 @@
+import sendgrid
+import os
+
+from sendgrid import Email, To, Content, Mail
+
+# Example 1
+# setting region to be "global"
+
+sg = sendgrid.SendGridAPIClient(api_key=os.environ.get('SENDGRID_API_KEY'))
+from_email = Email("example@abc.com")
+to_email = To("example@abc.com")
+subject = "Sending with SendGrid is Fun"
+content = Content("text/plain", "and easy to do anywhere, even with Python")
+print(sg.client.host)
```

Comment:
Do we want the print to be before or after setting the region?  In the example, this might be confusing because it would change if they set it to EU

Reply:
I might put the print statement in the same relative placement as example 2

Reply:
addressed